cni/network/invoker_cns.go

```diff
@@ -439,11 +439,15 @@ func configureDefaultAddResult(info *IPResultInfo, addConfig *IPAMAddConfig, add
 		if len(routes) > 0 {
 			resRoute = append(resRoute, routes...)
 		} else { // add default routes if none are provided
-			resRoute = append(resRoute, network.RouteInfo{
-				Dst: defaultRouteDstPrefix,
-				Gw:  ncgw,
-			})
+			// on Swiftv2 windows scenario, skip to add default route on infra vnet;
+			if !info.skipDefaultRoutes {
+				resRoute = append(resRoute, network.RouteInfo{
+					Dst: defaultRouteDstPrefix,
+					Gw:  ncgw,
+				})
+			}
```

Comment:
if routes are specified as part of cns goalstate why cni need to construct new one?

Reply:
what scenario this else case solves?

Reply:
it's not correct...I built image based on this code and did not push the correct codes.